CONTRIBUTING.md

```diff
@@ -21,6 +21,20 @@ phpize > /dev/null && \
 make clean > /dev/null && make all > /dev/null && make install
 ```
 
+The compiled extension will be generated into `./.libs/`.
+You can load the extension and check the version using the php command line.
+
+```
+php -d extension=$(pwd)/.libs/mongodb.so -r "echo (new ReflectionExtension('mongodb'))->getVersion();"
+```
+
+For larger tests, configure the extension in your `php.ini` file.
+
+```ini
+extension="<replace-with-absolute-project-path>/mongo-php-driver/.libs/mongodb.so"
+```
```

Comment:
What's the advantage of this over `make install`?

Reply:
Nothing. I missed that part of the command when running step by step.